apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -36,75 +38,92 @@ struct GroupMemberInfoView: View {
     }
 
     var body: some View {
-        NavigationView {
-            if let member = member {
-                List {
-                    groupMemberInfoHeader(member)
-                        .listRowBackground(Color.clear)
+        if navigation {
+            NavigationView { groupMemberInfoView() }
+        } else {
+            groupMemberInfoView()
+        }
+    }
+
+    private func groupMemberInfoView() -> some View {
+        VStack {
+            List {
+                groupMemberInfoHeader(member)
+                    .listRowBackground(Color.clear)
 
+                Section {
```

Comment:
how does it look if there's no send and no verify code button?

Reply:
though the second one should always be present..

Reply:
As nothing

Reply:
Maybe a bit more white space. Verify is only present for active members, not while they are connecting.

Reply:
Btw why we are showing send message for inactive?